[BrowserKit] Bump the dom-crawler minimum version requirement

jakzal opened the pull request:
| Q | A |
| --- | --- |
| Branch? | 2.3 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #18826 |
| License | MIT |
| Doc PR | - |

The BrowserKit Client will not work with DomCrawler 2.0 if the CssSelector component is not installed.

This will require an amendment when merging into the 2.8 branch. It currently uses `~2.0,>=2.0.5|~3.0.0`, while it should be `~2.1|~3.0.0`.

sstok commented:
@jakzal It seems your e-mail address is wrongly configured for Git, as your user linkage is broken for the commit.

jakzal committed:
[BrowserKit] Bump dom-crawler minimum version requirement

    The BrowserKit Client will not work with DomCrawler 2.0 if the CssSelector component is not installed.

jakzal commented:
@sstok oops. Thanks for letting me know. Fixed now :)

nicolas-grekas commented:
2.3 is not maintained anymore :)

jakzal commented:
@nicolas-grekas Old habbits die hard ;)

fabpot commented:
Thank you @jakzal.
